backend/src/main/java/com/carffeine/carffeine/car/infrastructure/repository/CarQueryRepository.java

```diff
@@ -0,0 +1,30 @@
+package com.carffeine.carffeine.car.infrastructure.repository;
+
+import com.carffeine.carffeine.car.controller.dto.CarResponse;
+import com.carffeine.carffeine.car.controller.dto.CarsResponse;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import static com.carffeine.carffeine.car.domain.QCar.car;
+import static com.querydsl.core.types.Projections.constructor;
+
+@RequiredArgsConstructor
+@Repository
+public class CarQueryRepository {
+
+    private final JPAQueryFactory jpaQueryFactory;
+
+    public CarsResponse findAllCars() {
+        return new CarsResponse(
+                jpaQueryFactory.select(
+                                constructor(CarResponse.class,
+                                        car.id,
+                                        car.name,
+                                        car.vintage
+                                )
+                        ).from(car)
+                        .fetch()
+        );
+    }
```

Comment:
```suggestion
    public List<CarResponse> findAllCars() {
        return jpaQueryFactory.select(
                                constructor(CarResponse.class,
                                        car.id,
                                        car.name,
                                        car.vintage
                                )
                        ).from(car)
                        .fetch();
                 }
```
이런식으로 반환하고 service에서 Carsresponse로 매핑하는 것이 좀 더 깔끔해보이는데 어떻게 생각하세요?